src/App.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import UserProfile from './components/UserProfile.jsx';
+import NotFoundPage from './containers/NotFoundPage';
+
+class App extends Component {
+  render() {
+    return (
+      <>
+        <NotFoundPage />
+        <UserProfile />
+      </>
```

Comment:
same with this line too, kindly check this

Reply:
That's the closing tag for the react fragment